workspaces/npm/plugins/npm/src/plugin.ts

```diff
@@ -28,3 +31,15 @@ export const npmPlugin = createPlugin({
     root: rootRouteRef,
   },
 });
+
+/** @public */
+export const NpmReleases = npmPlugin.provide(
```

Comment:
Can we use the same name as the component?

I think we should have *Card or *Table or *Page. (Maybe we should rename them to *EntityCard if they depend on an Entity context?)

Reply:
exporting it now with the exact component Name.